### ISSUE TITLE
foreach() argument error

### ISSUE THREAD

**Issue by joostvanhoof** [advoor/nova-editor-js#65]
I'm getting `foreach() argument must be of type array|object, string given` when saving the Nova resource with the EditorJS field.

Happens on line `25` of `NovaEditorJsData`:
```php
foreach ($attributes as $key => $value) {
  $this->attributes[$key] = $value;
}
```

Not sure if it's related but I'm using the `NovaEditorJsCast` on the model.

**Comment by joostvanhoof** [comment]
This solved my issue, but it's not ideal. It might provide some pointers to how to best fix it! @advoor:

I removed the cast `NovaEditorJsCast` from my model, and then used this in my Blade view:

`{{ \Advoor\NovaEditorJs\NovaEditorJs::generateHtmlOutput(json_decode($post->body)) }}`

This: `NovaEditorJs::generateHtmlOutput(json_decode($post->body))` (as per the docs) doesn't work.

**Comment by joostvanhoof** [comment]
@d3v4nt I think this is similar to [this issue](https://github.com/advoor/nova-editor-js/issues/58) that you posted?

**Comment by mennotempelaar** [comment]
@joostvanhoof 
I'm trying to get this `NovaEditorJsCast` to work within my model. I'm still receiving the `foreach() argument must be of type array|object, string given` error when I try to update the value.

When creating, it all works fine. 

The value is saved to the database:
![Screenshot 2022-07-13 at 17 03 38](https://user-images.githubusercontent.com/6905655/178766708-9b4e0f1f-9594-4dbc-9ef1-1560109a4228.png)
```
"{\"time\":1657723920527,\"blocks\":[{\"id\":\"P6zqsEgg0B\",\"type\":\"paragraph\",\"data\":{\"text\":\"test\"}}],\"version\":\"2.25.0\"}"
```

But when I want to update the value, it loads the edit page and when trying to add a second test to the first paragraph it returns the above error.

I've tried to find the solution but I keep getting stuck. The value is casted to an object, but for some reason it won't while updating.

I've added dumps within the `set` and `get` methods of the `NovaEditorJsCast` class and within the `__construct` of the `NovaEditorJsData` object.

This is the response from the xhr post request to `https://test-project.test/nova-api/post-resources/1?viaResource=&viaResourceId=&viaRelationship=&editing=true&editMode=update`:

![Screenshot 2022-07-13 at 17 08 44](https://user-images.githubusercontent.com/6905655/178767934-282d3bb0-1b83-4315-9329-ad11d9c8af6b.png)
![Screenshot 2022-07-13 at 17 09 19](https://user-images.githubusercontent.com/6905655/178767971-26192325-e233-42e3-9e0b-e2e59ebce34d.png)

Notice that there is not a single one with the double "test test" as `text` value.

I hope this gives some more insight into this problem. My database column is of type `LONGTEXT` and I've also tried `TEXT`.
